utils/backscrub.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+VERSION=20241106140722
+
+##
+## https://github.com/floe/backscrub
+##
+## backscrub, Fedora build script
+## Roman E. Chechnev <m0r1k>
+##
+
+ROOT_DIR=~/Downloads/backscrub_patched/
+BACKSCRUB_GIT_DIR=$ROOT_DIR/backscrub/
+BACKSCRUB_BUILD_DIR=$BACKSCRUB_GIT_DIR/build/
+CPU_CORES=$(grep -c processor /proc/cpuinfo)
+CAPTURE_DEV=/dev/video1
+VIRT_DEV_ID=0
+VIRT_DEV=/dev/video$VIRT_DEV_ID
+TARGET=/usr/local/bin/backscrub
+BACK_GROUND=$BACKSCRUB_GIT_DIR/backgrounds/rotating_earth.webm
+ACTION=$1
+
+set -ue;
```

Comment:
No `;` needed at the end of bash commands.

Reply:
yes, yes, ";" No need here, ";" No need there, and then in Kotlin you can’t do this:
Var str = "Test"
+ "Test"
+ "Test"